pkg/client/unversioned/client.go

```diff
@@ -28,6 +28,10 @@ import (
 	"k8s.io/kubernetes/pkg/version"
 )
 
+const (
+	clientGroupVersion = "v1"
```

Comment:
These are constants that logically belong to the high level client. I'd prefer to see a list here, e.g.:

```go
const (
  CoreGroupVersion = "v1"
  ExperimentalGroupVersion = "experimental/v1alpha1"
  ...
)
```

For extra points, the typesafe version would look something like:

```go

type GroupVersion string

const (
  Core = GroupVersion("v1")
  Experimental = GroupVersion("experimental/v1alpha1")
  ...
)
```

This is nice, but means you'd want to set the type of the parameter the Request.GroupVersion() function takes.